src/assets/VueOptionVsCompositionAPI/options-vs-composition-api.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Options Vs Composition API in Vue JS
+authorName: Sagar Gurung
+authorAvatar: https://avatars.githubusercontent.com/u/46086950?v=4
+authorLink: https://github.com/SagarGi
+createdAt: Jan 31, 2024
```

Comment:
Update date to the date when it actually gets published :)

Reply:
updated!